ivy/functional/frontends/paddle/tensor/random.py

```diff
@@ -21,17 +21,22 @@ def randint(low=0, high=None, shape=[1], dtype=None, name=None):
 
 
 @with_supported_dtypes(
-    {"2.5.0 and below": ("float32", "float64")},
+    {"2.4.2 and below": ("float32", "float64")},
     "paddle",
 )
 @to_ivy_arrays_and_back
 def poisson(x, name=None):
     return ivy.poisson(x, shape=None, device=None, dtype=None, seed=None, out=None)
 
 
-def randn(shape, dtype=None, name=None):
-    if dtype not in ["float32", "float64"]:
-        raise ivy.exceptions.IvyError(
-            "Unsupported dtype for randn, only float32 and float64 are supported, "
-        )
-    return ivy.random_normal(shape=shape, dtype=dtype, seed=None)
+def randn(shape, dtype=None, seed=0, name=None):
+    return ivy.random_normal(shape=shape, dtype=dtype, seed=seed)
```

Comment:
How is this related in this PR?

Reply:
My bad I accidentally rewrote on top of another function.